### PR TITLE
Update version to 1.5.0

### PR DESCRIPTION
**Pull request by friedger**
Use latest bintray release version to benefit from the new support for gradle plugin portal
Up version and changelog to 1.5.0

### CODE REVIEW

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/gradle-android-command-plugin-prb/4/
Test PASSed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 1.4.1
+- add Stop and Start task (contribution by to Sebastian Schuberth [#79](https://github.com/novoda/gradle-android-command-plugin/pull/79))
```
Comment (stefanhoth) [inline CHANGELOG.md]: Should read `contribution by [Sebastian Schuberth](https://github.com/sschuberth)` (with user link)

Comment (sschuberth) [inline CHANGELOG.md]: PR numbers are swapped here: PR #78 was the one adding the "Stop" task, PR #79 the one adding support for activity aliases.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+- add support fo launching via alias (`activity-alias`) (contribution by Sebastian Schuberth [#78](https://github.com/novoda/gradle-android-command-plugin/pull/78))
```
Comment (sschuberth) [inline CHANGELOG.md]: s/fo/for/

File: CHANGELOG.md (hunk 1 above)
Comment (stefanhoth) [inline CHANGELOG.md]: woops, thanks!

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/gradle-android-command-plugin-prb/5/
Test PASSed.

**Comment by friedger** [comment]
Let's wait for #83 to be merged before releasing the start action

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/gradle-android-command-plugin-prb/6/
Test PASSed.

**Comment by devisnik** [comment]
Shouldn't this be 1.5, since we have new functionality?

**Comment by friedger** [comment]
No, it should be 1.5.0 :-p. Issue updated accordingly.

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/gradle-android-command-plugin-prb/7/
Test PASSed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+# 1.4.1
```
Comment (stefanhoth) [inline CHANGELOG.md]: You wanted to change this to 1.5.0, right?

**Comment by stefanhoth** [comment]
Ready to merge this once we have the changelog fixed.

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/gradle-android-command-plugin-prb/8/
Test PASSed.

**Comment by devisnik** [comment]
:+1:

**Pull request merged**